docs/DeveloperGuide.md

```diff
@@ -707,6 +979,56 @@ testers are expected to do more *exploratory* testing.
     5. Test case: `edit 0 ua/10-10-2035`<br>
          Expected: No appointment is edited, as the index is invalid. Error details shown in the status message.
 
+### Deleting a patient's past appointment
+
+1. Deleting a patient's past appointment while all persons are being shown
+
+    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list. The first patient has 
+   at least one past appointment and the rest have zero.
+
+    3. Test case: `delappt 1`<br>
+       Expected: First patient's past appointment is deleted. Name of the deleted patient shown in the status message.
+
+    4. Test case: `delappt 2`<br>
+       Expected: No patient's past appointment. Error details shown in the status message.
+
+    5. Other incorrect delappt commands to try: `delappt`, `delappt x`, `...` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Consulting a patient
+
+1. Consulting a patient while all persons are being shown
+
+    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list. The second patient has
+       an upcoming appointment on the current day and the rest have no upcoming appointments.
+
+    2. Test case: `consult 1 diag/headache`<br>
+       Expected: First patient's has a past appointment created for the current day with a headache diagnosis.<br> 
+       Name of the consulted patient shown in the status message.
+
+    3. Test case: `consult 2 diag/headache`<br>
+       Expected: Second patient's upcoming appointment is deleted. The rest is similar to the above.
+
+    4. Other incorrect consult commands to try: `consult`, `consult x`, `...` (where x is larger than the list size)<br>
+       Expected: No past appointment is created. No upcoming appointment is removed. Error details shown in the status message.
+
+### View a patient
+
+1. View a patient while all persons are being shown
+
+    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list. The second patient has
+       an upcoming appointment on the current day and the rest have no upcoming appointments.
```

Comment:
do not think it is required for `The second patient has an upcoming appointment on the current day and the rest have no upcoming appointments` to be added

Reply:
This is for testing purposes to demonstrate that the upcoming appointment is reset using consult.